desktop_source/desktop_media_manager.cpp

```diff
@@ -41,21 +41,28 @@ int DesktopMediaManager::SourceRtpPort() const {
   return gst_pipeline_->UdpSourcePort();
 }
 
-std::vector<wfd::H264VideoFormat>
-DesktopMediaManager::SupportedH264VideoFormats() const {
-  return {format_};
-}
+std::vector<wfd::SelectableH264VideoFormat>
+DesktopMediaManager::GetSelectableH264VideoFormats() const {
+  std::vector<wfd::SelectableH264VideoFormat> formats;
+
+  wfd::RateAndResolution i;
+
+  for (i = wfd::CEA640x480p60; i <= wfd::CEA1920x1080p24; i = i + 1)
+      formats.push_back(wfd::SelectableH264VideoFormat(wfd::CHP, wfd::k4_2, wfd::CEARatesAndResolutions(i)));
```

Comment:
nit: IMO static_cast<wfd::CEARatesAndResolutions>(i) would be more readable, wfd::CEARatesAndResolutions(i) looks like a function call